database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -159,10 +160,30 @@ abstract class SlickDatabase(override val originalDatabaseConfig: Config) extend
     database.close()
   }
 
+  private val debugExitConfigPath = "danger.debug.only.exit-on-rollback-exception-with-status-code"
+  private val debugExitStatusCodeOption = ConfigFactory.load.getAs[Int](debugExitConfigPath)
+
   protected[this] def runTransaction[R](action: DBIO[R], isolationLevel: TransactionIsolation = TransactionIsolation.RepeatableRead): Future[R] = {
-    //database.run(action.transactionally) <-- See comment above private val actionThreadPool
-    val dbio = action.transactionally.withTransactionIsolation(isolationLevel)
-    Future(Await.result(database.run(dbio), Duration.Inf))(actionExecutionContext)
+    runAutoCommit(action.transactionally.withTransactionIsolation(isolationLevel))
```

Comment:
Discussed with faces, this seemed contradictory but is actually just a naming issue.